topics/chrome/index.md

```diff
@@ -0,0 +1,18 @@
+---
+aliases: google-chrome
+related: firefox,safari,edge,opera,chromium,browser,chrome-extension
+created_by: Google
+display_name: Chrome
+github_url: https://github.com/googlechrome
+logo: chrome.png
+released: September 2, 2008
+short_description: Chrome is a web browser from the tech company Google.
+topic: chrome
+url: https://www.google.com/chrome/
+---
+
+Chrome is the most popular web browser worldwide as of mid-2017, made by the tech company Google. It's available for most operating systems including Windows, macOS and Linux and on multiple platforms such as the desktop, phones and tablets.
+
+Chrome boasts a minimalistic UI and was the first browser to feature "tabs" above the address bar, a convention that was later implemented in other browsers. Other popular features include things such as Ingonito mode, Tab sandboxing and a Web Store to install extenions and themes from.
```

Comment:
Typo: `Ingonito` and `extenions`

I think "Tab" in "Tab sandboxing" should be lowercase.

Can you add a comma before `and a Web Store`?

Maybe instead of "Web Store to install extensions and themes from", "Web Store with extensions and themes".